src/olympia/reviewers/utils.py

```diff
@@ -789,7 +789,7 @@ def set_file(self, status, file):
 
     def set_promoted(self):
         group = self.addon.promoted_group(currently_approved=False)
-        if group and group.pre_review:
+        if self.version.channel == amo.CHANNEL_LISTED and group.pre_review:
```

Comment:
The `if group` is no longer needed?

Reply:
`Addon.promoted_group()` always returns a `PromotedClass` so it's safe.

(The old code worked because if the add-on isn't promoted `NOT_PROMOTED` is returned, which is falsey.)